pubsub.go

```diff
@@ -100,18 +100,25 @@ func ParseEvent[T any](e event.Event) (Request[T], error) {
 	return req, nil
 }
 
-func NewMockEvent[T any](manifest T, sender SenderType, action Action) (*event.Event, error) {
-	req := Request[T]{
+type MockEventOption[T any] func(*Request[T])
+
+func NewMockEvent[T any](manifest T, sender SenderType, action Action, options ...MockEventOption[T]) (*event.Event, error) {
+	req := &Request[T]{
 		ApiVersion: "orchestrator.entur.io/request/v1",
-		Metadata:   OuterMetadata{},
+		Metadata: OuterMetadata{
+			RequestID: "mockid",
```

Comment:
nice to have a value for quick and dirty tests, better than blank and forcing the option pattern everywhere